src/platform-includes/getting-started-config/javascript.capacitor.mdx

```diff
@@ -11,7 +11,20 @@ Sentry.init(
     // Set your release version, such as "getsentry@1.0.0"
     release: "my-project-name@<release-name>",
     // Set your dist version, such as "1"
-    dist: "<dist>"
+    dist: "<dist>",
+    integrations: [
+      // Registers and configures the Tracing integration,
+      // which automatically instruments your application to monitor its
+      // performance, including custom Angular routing instrumentation
+      new SentryAngular.BrowserTracing({
+        tracePropagationTargets: ["localhost", "https://yourserver.io/api"],
+        routingInstrumentation: Sentry.routingInstrumentation,
```

Comment:
Capacitor doesn't have `routingInstrumentation`
maybe you wanted to use it from the Angular package that contains it?
```suggestion
        routingInstrumentation: SentryAngular.routingInstrumentation,
```

Reply:
Update: 
not entirely sure what `routingInstrumentation: SentryAngular.routingInstrumentation` but seems like it stops registering transactions for page navigation when I set the routingInstrumentation

Reply:
> not entirely sure what routingInstrumentation: SentryAngular.routingInstrumentation

The Angular routing instrumentation is supposed to create transaction for page loads and navigations via the Angular Router. 

> seems like it stops registering transactions for page navigation
Are you using the Angular router? 
As I said I have no clue about capacitor so maybe this is a dumb question but: Any chance that routing works differently with Capacitor?

Reply:
Ahh I forgot that you have to provide `TraceService` to get navigation transactions. Maybe that's missing.